database/init.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const _ = require('lodash')
+const async = require('async')
+const pg = require('pg')
+const config = require('./../src/lib/config')
+
+if (config.get('env') !== 'test') {
```

Comment:
I'd allow this in a dev environment too: it would be useful as a way to quickly setup the db